manifests/package/debian.pp

```diff
@@ -14,12 +14,12 @@
 #
 # This class file is not called directly
 class nginx::package::debian (
-    $manage_repo              = true,
-    $package_name             = 'nginx',
-    $package_source           = 'nginx',
-    $package_ensure           = 'present',
-    $passenger_package_ensure = 'present'
-  ) {
+  $manage_repo              = true,
+  $package_name             = 'nginx',
+  $package_source           = 'nginx',
+  $package_ensure           = 'present',
+  $passenger_package_ensure = 'present'
+) {
```

Comment:
You're right, I misread this one, it's just indentation, so just fix that back to the original indentation and squash the commit.